packages/rx_bloc_cli/mason_templates/bricks/feature_onboarding/__brick__/lib/feature_onboarding_phone_confirmation/views/onboarding_phone_page.dart

```diff
@@ -20,54 +17,68 @@ class OnboardingPhonePage extends StatelessWidget {
   Widget build(BuildContext context) => RxForceUnfocuser(
         child: RxUnfocuser(
           child: Scaffold(
+            resizeToAvoidBottomInset: false,
```

Comment:
Please confirm this property is actually needed and it's well tested with small device sizes with a present keyboard

Reply:
Yes, I've tested with several different screen sizes (even custom ones with different DPIs) and included this chnage as a result of that.

Reply:
I think it causes issues with iPhone SE. I can not dismiss the keyboard when I tap on the text/icon
<img width="446" alt="image" src="https://github.com/user-attachments/assets/4012be8a-0f3d-417b-9355-186ba42f9fa1" />

Reply:
Addressed.